src/RefusjonProvider.tsx

```diff
@@ -14,20 +14,25 @@ export interface Context {
     feilListe: String[];
     setFeilListe: (feilListe: String[]) => void;
     ulagredeEndringer: boolean;
+
+    setLasterNå: (lasterNå: boolean) => void;
+    lasterNå: boolean;
 }
 
 export const RefusjonContext = React.createContext<Context>({} as Context);
 
 const RefusjonProvider: FunctionComponent = (props) => {
     const [refusjon, setRefusjon] = useState<Refusjon>({} as Refusjon);
+    const [lasterNå, setLasterNå] = useState(false);
     const [ulagredeEndringer, setUlagredeEndringer] = useState(false);
     const [feilListe, setFeilListe] = useState<String[]>([]);
-
     const { refusjonId } = useParams();
-    const nyRefusjon = useHentRefusjon(refusjonId);
-    if (refusjon !== nyRefusjon) {
-        setRefusjon(nyRefusjon);
-    }
```

Comment:
Fjernet henting og oppdatering av refusjon her. Bruker SWR useHentRefusjon for hver komponent